options/add_game.rb

```diff
@@ -0,0 +1,18 @@
+# Add a game to the list of games
+require './game'
+
+def add_game
+  puts 'Add a game:'
+  puts ''
+  puts 'Name:'
+  name = gets.chomp
+  puts 'Multiplayer (y/n):'
+  multiplayer = gets.chomp
+  puts 'Last played at:'
```

Comment:
- [ ] Kindly add more information to let the user know the kind of data to be entered and the format e.g `puts 'Enter last_played_at(yyyy-mm-dd)'`